american-journal-of-archaeology.csl

```diff
@@ -198,7 +198,7 @@
     <choose>
       <if type="article-journal">
         <text variable="volume" prefix=" "/>
-        <text variable="issue" prefix=" "/>
+        <text variable="issue" prefix=" (" suffix=")"/>
```

Comment:
where is that from? I'm actually not seeing issue numbers at all in the style guide

Reply:
No, that's correct, it is missing from the style guide. I'm looking at bibliography in articles from the journal for that. For example:

![Screen Shot 2021-11-15 at 10 11 14](https://user-images.githubusercontent.com/615115/141805705-092c82fc-c80e-4abe-b867-bf40bdca310b.png)